Tests/Indicators/TimeProfileTests.cs

```diff
@@ -117,11 +116,23 @@ public override void ResetsProperly()
             tp.Reset();
 
             TestHelper.AssertIndicatorIsInDefaultState(tp);
-            Assert.AreEqual(tp.POCPrice, 0m);
-            Assert.AreEqual(tp.POCVolume, 0m);
             tp.Update(new TradeBar() { Symbol = Symbols.IBM, Close = 1, Volume = 1, Time = reference.AddDays(1) });
             Assert.AreEqual(tp.Current.Value, 1m);
         }
+
+        [Test]
+        public override void WarmsUpProperly()
+        {
+            var tp = new TimeProfile(20);
+            var time = DateTime.Today;
```

Comment:
nit - Suggest replacing `DateTime.Today` for just creating a `DateTime(2020, 8, 1)`